requirements.txt

```diff
@@ -1,2 +1,2 @@
 requests >=2.7.0, <3.0.0
-
+mock==2.0.0
```

Comment:
We split out deplendencies between requirements.txt and test_requirements.txt.  The .travis.yml executes both (https://github.com/cloudant/python-cloudant/blob/master/.travis.yml#L14).  We already have a dependency for `mock` in https://github.com/cloudant/python-cloudant/blob/master/test-requirements.txt#L1.  I would remove this dependency.  I think even though the one in test_requirements.txt is 1.3.0, it should serve your needs.